apps/api/.env

```
@@ -0,0 +1,14 @@
+STRIPE_API_KEY=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)